src/core/connection.c

```diff
@@ -485,6 +485,27 @@ QuicConnUninitialize(
         QuicBindingRemoveConnection(Connection->Paths[0].Binding, Connection);
     }
 
+    CXPLAT_DBG_ASSERT(Connection->PathsCount >= 1);
+    QUIC_PATH* Path = &Connection->Paths[0];
+    if (Path->EncryptionOffloading) {
+        QUIC_CID_HASH_ENTRY* SourceCid =
+            CXPLAT_CONTAINING_RECORD(Connection->SourceCids.Next, QUIC_CID_HASH_ENTRY, Link);
+        CXPLAT_QEO_CONNECTION Offloads[2] = {0};
+        Offloads[0].Operation = CXPLAT_QEO_OPERATION_REMOVE;
+        Offloads[0].Direction = CXPLAT_QEO_DIRECTION_TRANSMIT;
+        Offloads[0].ConnectionIdLength = Path->DestCid->CID.Length;
+        Offloads[1].Operation = CXPLAT_QEO_OPERATION_REMOVE;
+        Offloads[1].Direction = CXPLAT_QEO_DIRECTION_RECEIVE;
+        Offloads[1].ConnectionIdLength = SourceCid->CID.Length;
+        memcpy(Offloads[0].ConnectionId, Path->DestCid->CID.Data, Path->DestCid->CID.Length);
+        memcpy(Offloads[1].ConnectionId, SourceCid->CID.Data, SourceCid->CID.Length);
+        if (!QUIC_SUCCEEDED(CxPlatSocketUpdateQeo(Path->Binding->Socket, Offloads, 2))) {
+            // TODO: Log
+        }
```

Comment:
```suggestion
        (void)CxPlatSocketUpdateQeo(Path->Binding->Socket, Offloads, 2);
```
Let's just make this stuff "best effort". I don't care much if it fails.